cloud: update docs for RBAC support (#14233)
pingcap/docs: tidb-cloud/migrate-from-amazon-s3-or-gcs.md

Before migrating data from GCS to TiDB Cloud, ensure the following:

- You have administrator access to your corporate-owned GCP account.

> - You have a TiDB Cloud account with at least the [`Project Data Access Read-Write`](/tidb-cloud/manage-user-access.md#user-roles) access.

Author: @Frank945946 PTAL

> - You have a TiDB Cloud account with at least the [`Project Data Access Read-Write`](/tidb-cloud/manage-user-access.md#user-roles) access.

Author: ```suggestion
- You have administrator access to the TiDB Cloud Management Portal.
```
